packages/esm-commons-lib/src/api/api.ts

```diff
@@ -9,6 +9,8 @@ import {
   covid_Assessment_EncounterUUID,
   covidOutcomesCohortUUID,
 } from '../constants';
+import { FhirPatient, FhirPatientResponse, PatientListRow } from './types';
+import useSWR from 'swr';
```

Comment:
Do you need this?

Reply:
This is residue after moving the new endpoint to another file and need to be removed 👍🏾